composeml/label_times.py

```diff
@@ -22,20 +30,44 @@ def __init__(self, data=None, name=None, target_entity=None, settings=None, tran
         self.transforms = transforms or []
         self.plot = LabelPlots(self)
 
+        if label_type is not None:
+            error = 'label type must be "continuous" or "discrete"'
+            assert label_type in ['continuous', 'discrete'], error
+
+        if label_type is None and name in self.columns:
```

Comment:
why would `name` not be in `self.columns`?

Reply:
Came across behavior where pandas would initialize label times without passing value for `name`. I refactored logic to infer inside `is_discrete` only if label type is none. [link](https://github.com/FeatureLabs/compose/pull/62/files#diff-5d06a8d5b7b55549065b9c8f90f45edfR47-R48)